pytorch_pfn_extras/onnx/export_testcase.py

```diff
@@ -119,7 +120,10 @@ def _export_util(
     try:
         torch.onnx.utils._model_to_graph = _model_to_graph_with_value_names
         return torch_export(  # type: ignore[no-untyped-call]
-            model, args, f, _retain_param_name=True, **kwargs)
+            model, args, f, **kwargs)
+    except torch.onnx.utils.ONNXCheckerError:
```

Comment:
This exception class is added in 1.10: https://github.com/pytorch/pytorch/pull/61708/files#diff-849a5778e2dcf7f36587967273cee0bf20642e35bf4c79405111ea3417c3fb3cR28

Reply:
yes, I am adding version checks that works around this and other problems